maestro-cli/src/main/java/maestro/cli/update/Updates.kt

```diff
@@ -46,6 +65,24 @@ object Updates {
         }
     }
 
+    private fun fetchChangelog(): List<String>? {
+        if (CLI_VERSION == null) {
+            return null
+        }
+        val request = Request.Builder()
+            .url("https://raw.githubusercontent.com/mobile-dev-inc/maestro/main/CHANGELOG.md")
+            .build()
+        val content = OkHttpClient().newCall(request).execute().body?.string()
+        val body = content?.split("\n## ")?.map { it.lines() }
+        val version = fetchUpdates()?.toString() ?: return null
+        return body
+            ?.first { it.first().startsWith(version) }
+            ?.drop(1)
+            ?.map { it.trim().replace("**", "") }
+            ?.map { it.replace("\\[(.*?)]\\(.*?\\)".toRegex(), "$1") }
+            ?.filter { it.isNotEmpty() && it.startsWith("- ") }
+    }
+
```

Comment:
I'd hesitant to merge this code. We'll have to be very careful with format of our changelog.

Could we [use CommonMark instead, and use its basic `TextContentRenderer`?](https://github.com/commonmark/commonmark-java?tab=readme-ov-file#render-to-markdown)